website/docs/r/binaryauthorization_attestor.html.markdown

```diff
@@ -23,17 +23,14 @@ description: |-
 
 An attestor that attests to container image artifacts.
 
-~> **Warning:** This resource is in beta, and should be used with the terraform-provider-google-beta provider.
```

Comment:
Hmm, it would be better if these could be kept. How much does that screw things up?